container/shim/src/index.js

```diff
@@ -89,7 +104,7 @@ const handleCID = asyncHandler(async (req, res) => {
 
   res.set({
     'Content-Type': mimeTypes.lookup(req.path) || 'application/octet-stream',
-    'Cache-Control': 'public, max-age=31536000, immutable',
+    'Cache-Control': 'public, max-age=29030400, immutable',
```

Comment:
Why lower it?

Reply:
Just so the tests pass. There's not a significant difference between 336 days and 365 days anyhow.

https://github.com/ipfs/kubo/blob/master/test/sharness/t0116-gateway-cache.sh#L74
https://github.com/ipfs/kubo/blob/master/test/sharness/t0117-gateway-block.sh#L71
https://github.com/ipfs/kubo/blob/master/test/sharness/t0118-gateway-car.sh#L132

Reply:
I wonder why they chose 336 days, but yeah, it is fine